packages/lesswrong/components/tagging/subforums/SubforumWikiTab.tsx

```diff
@@ -0,0 +1,166 @@
+import classNames from 'classnames';
+import React, { useCallback, useState } from 'react';
+import { useLocation } from '../../../lib/routeUtil';
+import { Components, registerComponent } from '../../../lib/vulcan-lib';
+import { AnalyticsContext, useTracking } from "../../../lib/analyticsEvents";
+import { EditTagForm } from '../EditTagPage';
+import { useApolloClient } from '@apollo/client/react';
+import truncateTagDescription from "../../../lib/utils/truncateTagDescription";
+import { forumTypeSetting, taggingNamePluralSetting } from '../../../lib/instanceSettings';
+import { truncate } from '../../../lib/editor/ellipsize';
+import { tagPostTerms } from '../TagPage';
+import { useOnSearchHotkey } from '../../common/withGlobalKeydown';
+import { MAX_COLUMN_WIDTH } from '../../posts/PostsPage/PostsPage';
+import { tagMinimumKarmaPermissions, tagUserHasSufficientKarma } from '../../../lib/collections/tags/helpers';
+import { useCurrentUser } from '../../common/withUser';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  centralColumn: {
+    marginLeft: "auto",
+    marginRight: "auto",
+    maxWidth: MAX_COLUMN_WIDTH,
+  },
+  tagHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+  },
+  postsTaggedTitle: {
+    color: theme.palette.grey[600]
+  },
+  pastRevisionNotice: {
+    ...theme.typography.commentStyle,
+    fontStyle: 'italic'
+  },
+  wikiSection: {
+    paddingTop: 12,
+    paddingLeft: 42,
+    paddingRight: 42,
+    paddingBottom: 12,
+    marginBottom: 24,
+    background: theme.palette.panelBackground.default,
+  },
+})
+
+const isEAForum = forumTypeSetting.get() === 'EAForum'
+
+const SubforumWikiTab = ({tag, revision, truncated, setTruncated, classes}: {
+  tag: TagPageFragment | TagPageWithRevisionFragment,
+  revision?: string,
+  truncated: boolean,
+  setTruncated: (truncated: boolean) => void,
+  classes: ClassesType,
+}) => {
+  const {
+    PostsListSortDropdown,
+    PostsList2,
+    ContentItemBody,
+    AddPostsToTag,
+    UsersNameDisplay,
+    TagDiscussionSection,
+    TagPageButtonRow,
+    TagIntroSequence,
+    SectionTitle,
+    ContentStyles,
+  } = Components;
+
+  const currentUser = useCurrentUser();
+  const { query } = useLocation();
+  const client = useApolloClient()
+  const { captureEvent } =  useTracking()
+  const terms = {
+    ...tagPostTerms(tag, query),
+    limit: 15
+  }
+  
+  const [editing, setEditing] = useState(!!query.edit || !!query.flagId)
```

Comment:
This is a behaviour change. Previously if there was a `flagId` in the url `editing` would be false to begin with, and only be set to true if you click "next" to go to the next tag. I changed this to get ~almost the same behaviour while avoiding having to pass `editing, setEditing` down from TagSubforumPage2